From b36d0ff6350ac4ff38459351396125238fa8df92 Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Mon, 21 Nov 2011 16:24:30 -0500
Subject: [PATCH 136/256] mtd: add device-id for numonyx nor flash

Added device-id definition for Numonix SPI NOR flash device.

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
(cherry picked from commit f3f13824281b79c29790ec541ad3da9165ecfed8)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/mtd/devices/m25p80.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/mtd/devices/m25p80.c b/drivers/mtd/devices/m25p80.c
index 524dab3..bdaacd7 100644
--- a/drivers/mtd/devices/m25p80.c
+++ b/drivers/mtd/devices/m25p80.c
@@ -999,6 +999,9 @@ static const struct spi_device_id m25p_ids[] = {
 	{ "m25p128", INFO(0x202018,  0, 256 * 1024,  64, 0) },
 	{ "n25q032", INFO(0x20ba16,  0,  64 * 1024,  64, 0) },
 
+	/* Numonyx */
+	{ "n25q032",  INFO(0x20bb16,  0,  64 * 1024,  64, 0) },
+
 	{ "m25p05-nonjedec",  INFO(0, 0,  32 * 1024,   2, 0) },
 	{ "m25p10-nonjedec",  INFO(0, 0,  32 * 1024,   4, 0) },
 	{ "m25p20-nonjedec",  INFO(0, 0,  64 * 1024,   4, 0) },
-- 
1.7.5.4

